orange/src/main/java/com/hartwig/hmftools/orange/report/tables/SignatureAllocationTable.java

```diff
@@ -33,13 +34,15 @@ public static Table build(@NotNull String title, float width, @NotNull List<Sign
 
         Cells cells = new Cells(reportResources);
         Table table = Tables.createContent(width,
-                new float[] { 1, 1, 1, 3 },
-                new Cell[] { cells.createHeader("Signature"), cells.createHeader("Allocation"), cells.createHeader("Percent"),
+                new float[] { 1, 3, 1, 1, 1 },
+                new Cell[] { cells.createHeader("Signature"), cells.createHeader("Etiology"), cells.createHeader("Allocation"),
+                        cells.createHeader("Percent"),
                         cells.createHeader(Strings.EMPTY) });
 
         for(SignatureAllocation signatureAllocation : sort(signatureAllocations))
         {
             table.addCell(cells.createContent(signatureAllocation.signature()));
+            table.addCell(cells.createContent(Objects.requireNonNullElse(signatureAllocation.etiology(), "-")));
```

Comment:
As mentioned before, I would set "-" for MISALLOC in the algo and make the etiology itself NotNull